backend/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

There is a convention about npm packages:
Running `npm start` should start a package.

```suggestion
    "start": "node app.js"
```

https://docs.npmjs.com/cli/v8/commands/npm-start